### DISCUSSION TITLE
[GENERAL] Something to help testers / Phenomenal cosmic powers

### DISCUSSION THREAD

**Discussion by sluggor** [azerothcore/azerothcore-wotlk#3915] [General]
Sometimes I need to run through somewhere or kill something repeatedly to test something, and I can't always use GM commands to test properly, so I made a little ring that helps me not only get through any necessary combat, but also allows me to wear pretty much any armor I want, including the trash from the starter areas, when killing just about anything.

It's like a bargain version of transmogs.

Anyway, if anyone is interested, here's the ring I've been using to breeze through my own testing, you may find it useful. It replaces a test ring that doesn't seem to be used anywhere.

![ro1000t](https://user-images.githubusercontent.com/50815505/58372341-a0591200-7ed0-11e9-8560-a0683d0484b4.jpg)

After you run the SQL below against your database, just use `.additem 996` with either no target or your character targeted to receive the ring.

```SQL
USE `acore_world`;

-- Delete old test ring

DELETE FROM `item_template` WHERE `entry` = 996;

-- Insert new Ring of a Thousand Truths

INSERT INTO `item_template`(`entry`, `class`, `name`, `displayid`, `Quality`, `Flags`, `BuyPrice`, `SellPrice`, `InventoryType`, `ItemLevel`, `RequiredLevel`, `StatsCount`, `stat_type1`, `stat_value1`,  `stat_type2`, `stat_value2`,  `stat_type3`, `stat_value3`,  `stat_type4`, `stat_value4`,  `stat_type5`, `stat_value5`,  `stat_type6`, `stat_value6`,  `stat_type7`, `stat_value7`,  `stat_type8`, `stat_value8`,  `stat_type9`, `stat_value9`,  `stat_type10`, `stat_value10`, `delay`, `description`, `Material`, `ItemLimitCategory`, `DisenchantID`, `VerifiedBuild`) VALUES
(996, 4, 'Ring of a Thousand Truths', 3453, 4, 524288, 500000, 50000, 11, 300, 1, 10, 1, 32767, 3, 32767, 4, 32767, 5, 32767, 7, 32767, 36, 32767, 38, 32767, 45, 32767, 43, 32767, 46, 32767, 0, 'Just remember, it\'s not a lie if you believe it.', 5, 67, 68, 15595);
```

**Comment by BarbzYHOOL** [comment]
It's just a super powerful ring? it can be handy in certain situation though so that's a good one.
Are you on the discord? Because I have some plans for small stuff like that

**Comment by sluggor** [comment]
Hi! I am on discord, I'm just not very good at. What kind of plans are you thinking?

**Comment by BarbzYHOOL** [comment]
These kind of tools could as well be a reward for contributors

**Comment by BarbzYHOOL** [comment]
Also `.add 17` (martin's fury)
